packages/node_modules/brightspace-auth-keys-dynamodb-store/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@d2l/node-auth-provisioning-dynamodb",
+  "version": "1.0.0",
+  "description": "DynamoDB based public key store implementation for brightspace-auth-keys",
+  "main": "src/public-key-store.js",
+  "scripts": {
+    "test": "nyc --all mocha --timeout 15000 --recursive ./test && npm run check-style",
+    "check-style": "eslint ."
+  },
+  "author": "D2L Corporation",
+  "license": "Apache-2.0",
+  "devDependencies": {
```

Comment:
Most of these should already [installed at the top level](https://github.com/Brightspace/node-auth-keys/blob/master/package.json#L19).

Trying to keep most things there, since they'll be mostly re-used, and greenkeeper will manage them that way.

I did the the peerDep for "redis" at this level, but i dunno if that was the right move. The scripts don't handle that yet though, so definitely keep that here.